apps/journeys-admin/src/components/Editor/Fab/Fab.spec.tsx

```diff
@@ -0,0 +1,141 @@
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { fireEvent, render, waitFor } from '@testing-library/react'
+
+import {
+  ActiveContent,
+  ActiveFab,
+  EditorProvider,
+  EditorState
+} from '@core/journeys/ui/EditorProvider'
+import {
+  ActiveCanvasDetailsDrawer,
+  ActiveSlide
+} from '@core/journeys/ui/EditorProvider/EditorProvider'
+
+import { TestEditorState } from '../../../libs/TestEditorState'
+
+import { Fab } from './Fab'
```

Comment:
test the index file here too.

```suggestion
import { Fab } from '.'
```